boilerplate/openshift/golang-osd-operator/csv-generate/csv-generate.sh

```diff
@@ -33,13 +35,27 @@ while getopts "c:dg:H:i:n:o:" option; do
         o)
             operator_name=${OPTARG}
             ;;
+        V)
```

Comment:
Capital for this parameter seems odd.

Reply:
I feel like `-v` is universally reserved for either "verbose" or "version". I had a similar thing with `-h` (always "help") on @bdematte's PR, which is why we ended up with `-H` :)